doc/source/development/contributing_codebase.rst

```diff
@@ -828,3 +835,10 @@ This will put the text *New in version 1.1.0* wherever you put the sphinx
 directive. This should also be put in the docstring when adding a new function
 or method (`example <https://github.com/pandas-dev/pandas/blob/v0.20.2/pandas/core/frame.py#L1495>`__)
 or a new keyword argument (`example <https://github.com/pandas-dev/pandas/blob/v0.20.2/pandas/core/generic.py#L568>`__).
+
+If your code is a bugfix, add your entry to the relevant bugfix section. Avoid
+adding to the ``Other`` section; only in rare cases should entries go here.
```

Comment:
Maybe it's just me, but the `here` in this sentence feels a bit confusing and unnatural. I'd refer to the `Other` section as `this section`, or more explicity, saying the `'Other' section` again.

Reply:
Agreed that the use of here is not clear - what do you think of "there" instead?